iceberg/iceberg-common/build.gradle.kts

```diff
@@ -41,17 +48,22 @@ dependencies {
     exclude("com.github.spotbugs")
     exclude("com.sun.jersey")
     exclude("javax.servlet")
+    exclude("org.apache.curator")
+    exclude("org.apache.zookeeper")
     exclude("org.mortbay.jetty")
   }
+  // use hdfs-default.xml
   implementation(libs.hadoop2.hdfs) {
```

Comment:
Do we still need this, is hdfs-client enough?

Reply:
I failed to exclude `hadoop-hdfs` jar because we depends`hdfs-default.xml` to initialize the related environment.